tasks/shibbolethsp_edugain.yml

```diff
@@ -35,3 +35,15 @@
                  type: "RequireValidUntil"
                  maxValidityInterval: "2592000"
 
+- name: Shibboleth SP | eduGAIN | clean MetadataFilter elements if shibbolethsp_edugainmd_skipfilters is set |
+  xml:
```

Comment:
Currently SP role seems to miss:

`
<MetadataFilter type="EntityRoleWhiteList">
  <RetainedRole>md:IDPSSODescriptor</RetainedRole>
</MetadataFilter>
`
Should we add this? If so then xpath shoud target more accurately to correct filter.
But maybe that's future improvements, is this tested?